### ISSUE TITLE
mail.json Cloudformation template aborts when attempting to create a fresh stack.

### ISSUE THREAD

**Issue by greendavegreen** [hybox/aws#104]
I create a new account for hosting a hybox deployment.  I invoked using the templates from the repository and our own parameters ( 1 keypair, 1 hosted zone, Rails secret, and database user/PW)

```aws aws --region $AWS_DEFAULT_REGION cloudformation create-stack --stack-name hybox --template-body https://s3.amazonaws.com/hybox-deployment-artifacts/cloudformation/current/templates/stack.json --capabilities CAPABILITY_IAM --parameters file://params/iter1.json```

mail.json aborts during creating a new hybox deployment in our brand new AWS account.  The complaint in the Events tab of the stack mentions:

> The runtime parameter of nodejs is no longer supported for creating or updating AWS Lambda functions. We recommend you use the new runtime (nodejs4.3) while creating or updating functions.

looking at the mail.json template, there are multiple lambda functions being declared and all of them use the (now illegal) value of 'nodejs' for their requested runtime.  I'm not fluent enough in javascript or the intent of the lambdas to understand if just flipping the runtime to the new version is safe/compatible.

the pertinent info from the AWS docs:  
```To use the Node.js runtime v4.3, set the value to "nodejs4.3". To use earlier runtime (v0.10.42), set the value to "nodejs".```

```Note
You can no longer create functions using the v0.10.42 runtime version as of November, 2016. Existing functions will be supported until early 2017, but we recommend you migrate them to nodejs4.3 runtime version as soon as possible.```


perhaps this only happens with new installs and thus wouldn't be causing those with existing deployments pain yet?

**Comment by mjgiarlo** [comment]
Heads-up, @eefahy @cbeer

**Comment by eefahy** [comment]
Closed by 11cae893273f1d57edf43bddaa8b75506247fc82

**Comment by mjgiarlo** [comment]
Thanks for raising this, @greendavegreen!
